modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -222,7 +222,9 @@ public ObjectDefinition postObjectDefinition(
 			throw new ObjectDefinitionStorageTypeException();
 		}
 
-		_addListTypeDefinition(objectDefinition);
+		if (GetterUtil.getBoolean(PropsUtil.get("feature.flag.LPS-164278"))) {
+			_addListTypeDefinition(objectDefinition);
+		}
```

Comment:
@murilostodolni not sure if we need this feature flag here, since `ObjectFieldUtil.addListTypeDefinition` is already under feature flag.

Reply:
Perfect!